src/Microsoft.ML.KMeansClustering/KMeansPlusPlusTrainer.cs

```diff
@@ -90,8 +90,7 @@ internal static class Defaults
         }
 
         /// <summary>
-        /// Options for the <see cref="KMeansTrainer"/> as used in
-        /// [KMeansTrainer(Options)](xref:"Microsoft.ML.KMeansClusteringExtensions.KMeans(Microsoft.ML.ClusteringCatalog.ClusteringTrainers,Microsoft.ML.Trainers.KMeansTrainer.Options)".
+        /// Options for the <see cref="KMeansTrainer"/> as used in [KMeansTrainer(Options)](xref:Microsoft.ML.KMeansClusteringExtensions.KMeans(Microsoft.ML.ClusteringCatalog.ClusteringTrainers,Microsoft.ML.Trainers.KMeansTrainer.Options)).
```

Comment:
>[KMeansTrainer(Options)](xref:Microsoft.ML.KMeansClusteringExtensions.KMeans(Microsoft.ML.ClusteringCatalog.ClusteringTrainers,Microsoft.ML.Trainers.KMeansTrainer.Options) [](start = 67, length = 171)

any reason why you not using `<see cref="Microsoft.ML.Trainers.KMeansTrainer.Options()"/> here?
I though only reason to use xref is then they inside markdown there usual `<see>` tags don't work.

Reply:
@shmoradims said xref is desired because it can shorten the name of reference. Using xref is a explicit request from him.